java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S5803_java.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "Class members annotated with \"@VisibleForTesting\" should not be accessed from production code ",
```

Comment:
There is a suspicious extra space at the end of the title. You might want to check on Jira.

Reply:
In the ticket title in Jira there is a space at the end. Will fix on both sides